websites/N/Night.design/presence.ts

```diff
@@ -0,0 +1,119 @@
+const presence = new Presence({
+	clientId: "1031730262092042321",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/4mDvKNa.png",
+	};
+
+	presenceData.startTimestamp = Math.floor(Date.now() / 1000);
+
+	const pathname = document.location.pathname.slice(3);
+
+	switch (pathname) {
+		case "/home":
+			presenceData.details = "View the Homepage";
+			break;
+		case "/resources":
+			presenceData.details = "Search...";
+			break;
+		case "/settings/account":
+			presenceData.details = "Change my Account Settings";
+			break;
+		case "/settings/notifications":
+			presenceData.details = "Change my Notifications Settings";
+			break;
+		case "/settings/connected-accounts":
+			presenceData.details = "Add a user Connection to my account";
+			break;
+		case "/settings/account-security":
+			presenceData.details = "Change my Account Security Settings";
+			break;
+		case "/password-forgotten":
+			presenceData.details = "Change my Password";
+			break;
+		case "/login":
+			presenceData.details = "Sign in with my Account";
+			break;
+		case "/register":
+			presenceData.details = "Create a new Account";
+			break;
+		case "/dashboard":
+			presenceData.details = "View my Dashboard";
+			break;
+		case "/creator":
+			presenceData.details = "View Creator page";
+			break;
+		case "/modder":
+			presenceData.details = "View Modder Page";
+			break;
+		case "/modder/publisher":
+			presenceData.details = "Apply to become a Modder";
+			break;
+		case "/legal/terms-of-service":
+			presenceData.details = "Read the Terms of Service from Night.design";
+			break;
+		case "/legal/privacy-policy":
+			presenceData.details = "Read the Privacy Policy from Night.design";
+			break;
+		case "/legal/imprint":
+			presenceData.details = "Read the Imprint from Night.design";
+			break;
+		case `/resources/${document.location.pathname.split("/")[3]}`:
+			{
+				const pic = document.querySelector(".img-fluid") as HTMLImageElement;
+				presenceData.largeImageKey = pic.src;
+				presenceData.details = document.querySelector(".m-b-10").innerHTML;
+			}
+			break;
+		case `/u/${document.location.pathname.split("/")[3]}/activities`:
+			{
+				const image = document.querySelectorAll(
+					".rounded-circle"
+				)[0] as HTMLImageElement;
```

Comment:
These can be simplified to something like the following.
```suggestion
				const image = document.querySelector<HTMLImageElement>(
					".rounded-circle"
				);
```